moto/core/models.py

```diff
@@ -766,6 +766,47 @@ def get_config_resource(
         """
         raise NotImplementedError()
 
+    def aggregate_regions(self, path, backend_region, resource_region):
+        """
+        This method is called for both aggregated and non-aggregated calls for config resources.
+
+        It will figure out how to return the full list of resources for a given regional backend and append them to a final list.
+        It produces a list of both the region and the resource name with a delimiter character (CONFIG_BACKEND_DELIM, ASCII Record separator, \x1e).
+        IE: "us-east-1\x1ei-1234567800"
+
+        You should only use this method if you need to aggregate resources over more than one region.
```

Comment:
This function was used previously in your code for both aggregation and non-aggregation contexts. Perhaps rename the function? (or remove it?)

Reply:
I removed it for now; I can add it back when I do something that's truly "regional". This whole PR has been a bit of a learning journey.